src/main/java/com/google/longrunning/OperationsClient.java

```diff
@@ -47,17 +47,15 @@
 /**
  * Service Description: Manages long-running operations with an API service.
  *
- * <p>
- * When an API method normally takes long time to complete, it can be designed to return
+ * <p>When an API method normally takes long time to complete, it can be designed to return
```

Comment:
I think all of these changes are due to a difference in the formatter. I'm not sure whether your changes are correct, or should be reverted...